backend/src/jwt/decode.spec.js

```diff
@@ -69,23 +69,23 @@ describe('decode', () => {
           {
             role: 'user',
             name: 'Jenny Rostock',
-            avatar: 'https://s3.amazonaws.com/uifaces/faces/twitter/sasha_shestakov/128.jpg',
             id: 'u3',
             slug: 'jenny-rostock',
           },
           {
+            image: Factory.build('image', {
+              url: 'https://s3.amazonaws.com/uifaces/faces/twitter/sasha_shestakov/128.jpg',
+            }),
             email: 'user@example.org',
           },
         )
       })
 
-      it('returns user object except email', async () => {
+      it('returns user object without email', async () => {
         await expect(decode(driver, authorizationHeader)).resolves.toMatchObject({
           role: 'user',
           name: 'Jenny Rostock',
-          avatar: 'https://s3.amazonaws.com/uifaces/faces/twitter/sasha_shestakov/128.jpg',
           id: 'u3',
-          email: null,
           slug: 'jenny-rostock',
         })
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-20T19:41:32Z" title="Thursday, February 20th 2020, 8:41:32 pm +01:00">Feb 20, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -69,24 +69,25 @@ describe('decode', () => {
           {
             role: 'user',
             name: 'Jenny Rostock',
-            avatar: 'https://s3.amazonaws.com/uifaces/faces/twitter/sasha_shestakov/128.jpg',
             id: 'u3',
             slug: 'jenny-rostock',
           },
           {
+            image: Factory.build('image', {
+              url: 'https://s3.amazonaws.com/uifaces/faces/twitter/sasha_shestakov/128.jpg',
+            }),
             email: 'user@example.org',
           },
         )
       })
 
-      it('returns user object except email', async () => {
+      it('returns user object without email', async () => {
         await expect(decode(driver, authorizationHeader)).resolves.toMatchObject({
           role: 'user',
           name: 'Jenny Rostock',
-          avatar: 'https://s3.amazonaws.com/uifaces/faces/twitter/sasha_shestakov/128.jpg',
           id: 'u3',
-          email: null,
           slug: 'jenny-rostock',
+          // email: undefined,
```


?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-22T12:40:57Z" title="Saturday, February 22nd 2020, 1:40:57 pm +01:00">Feb 22, 2020</time>_

---


The `authorizationHeader` does not contain the email anymore. It's a small code creep when I removed `.avatar` from the list of attributes passed to `encode`.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-25T14:33:46Z" title="Tuesday, February 25th 2020, 3:33:46 pm +01:00">Feb 25, 2020</time>_

---


sorry, my question is why this is commented out? can you just remove it?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-03-02T14:48:55Z" title="Monday, March 2nd 2020, 3:48:55 pm +01:00">Mar 2, 2020</time>_

---


I'm going to remove. I thought it could be a hint that `email` is not included in the token.